pkg/kubelet/apis/config/types.go

```diff
@@ -656,3 +661,16 @@ type MemorySwapConfiguration struct {
 	// +optional
 	SwapBehavior string
 }
+
+// RateLimitConfiguration is used for setting the rate limits for the kubelet APIs exposed
+// locally on the node as gRPC over UNIX-domain sockets, like the podresources API
+type RateLimitConfiguration struct {
+	// MaxFrequency  defines the maximum allowed frequency of the calls represented
+	// as number of calls per second, cumulating all the calls.
+	// Omitting the value or setting to zero means no limit.
+	MaxFrequency float64
```

Comment:
how should admins tune this limit? we're generally trying to move away from fixed qps limits on other clients/servers since they're difficult to administer (in favor of things like priority-and-fairness in the API server)